### DISCUSSION TITLE
How to write an addon with Vue?

### DISCUSSION THREAD

**Discussion by isorna** [storybookjs/storybook#25650] [Help]
### Summary

I need to create some addons for my Storybook implementation and there seems that it's only possible to create them with the current [https://storybook.js.org/docs/addons/writing-addons](react documentation) and [https://github.com/storybookjs/addon-kit](Addon Kit).

Did someone figured out how to write Storybook addons with Vue?, I'd like to preserve the project's code as uniform as possible, and its libraries if possible.

**Comment by Marklb** [comment]
Storybook is written in React, so addons have to use React. There is nothing stopping you from mixing frameworks and rendering Vue components in a React component, but it will still be using React at it's core.

I don't know Vue, so I can't provide an example for how to use Vue in a React component, but there are probably non-Storybook examples for doing that, if you can't find a Storybook addon that is already doing it.

**Comment by isorna** [comment]
I see, I don't mind using vanilla JS (in fact I prefer it), it should be possible to do so isn't it?
